tests/crypto/crl_tests.c

```diff
@@ -127,6 +127,19 @@ static bool _year_is_a_leap_year(uint32_t year)
     }
 }
 
+static int32_t _diff_oe_datetime(oe_datetime_t* t1, oe_datetime_t* t2)
+{
+    int32_t dyear = (int32_t)t1->year - (int32_t)t2->year;
+    int32_t dmonth = (int32_t)t1->month - (int32_t)t2->month;
```

Comment:
For a correct `diff` of date time won't there be logic to handle the varying lengths of different months, leap years etc?

Reply:
Actually that's a great point.  A safer algorithm would be to convert to time_t (using mktime or whatever) and then take the difference in seconds.